docs/src/test-api/class-testconfig.md

```diff
@@ -439,6 +439,14 @@ export default defineConfig({
 
 Test files that took more than `threshold` milliseconds are considered slow, and the slowest ones are reported, no more than `max` number of them. Passing zero as `max` reports all test files that exceed the threshold.
 
+## property: TestConfig.respectGitIgnore
+* since: v1.45
+- type: ?<[boolean]>
+
+Whether to skip entries from `.gitignore` when searching for test files. By default it is true, i.e. if
```

Comment:
How is `retries` related to `.gitignore`?

Reply:
It's `entries` not `retries`